### DISCUSSION TITLE
Some additional /find functions

### DISCUSSION THREAD

**Discussion by CaptainRexPL** [Pilot1782/bad_copeheimer#98] [Ideas]
**Is your feature request related to a problem? Please describe.**
No
**Describe the solution you'd like**
1. Possibility of looking for servers from your database with motd flag.
2. Make /find (version) command print all the servers with the version and not only one random server from the DB.

**Comment by Pilot1782** [comment]
I think limiting the number of server it responds with would be better as you wouldn't want to be spammed with 300 servers that match your description.

MOTD should be pretty easy to add, I'll do that soon.

**Reply by Pilot1782** [reply]
MOTD is added along with tweaks to how version works, let me know if you like it.

**Reply by CaptainRexPL** [reply]
It still doesn't print multiple servers with the same MOTD, but it works as intended so finding a server with unique MOTD is now possible. Thank you for implementing this. 
Is it possible to implement searching for servers with specific number of max players and add a possibility for the bot to print the server image as well?

**Reply by Pilot1782** [reply]
Adding max players should be fine but as for the icon I've been trying that for awhile and not found anything yet, I'll keep looking but it might be a while.

**Comment by Pilot1782** [comment]
![image](https://user-images.githubusercontent.com/71522520/210124692-58423d2d-7990-46ae-a898-b4b118890417.png)
Added Max Players but right now its only returning 1 server at a time

current args
- _id
- host
- port (with host)
- player (doesn't really work rn)
- version
- motd
- max players

**Reply by Pilot1782** [reply]
As of the latest dev-builds commit, server icons now work.
